articles/user-guide/libraries/index.md

```diff
@@ -17,5 +18,9 @@ In this section of the documentation, we describe these libraries and how to use
   This section describes the quantum chemistry library, which provides a data model for loading representations of fermionic Hamiltonians and quantum simulation operations and functions which act on these representations.
 - [**Quantum numerics library**](xref:microsoft.quantum.numerics.intro):
   This section describes the quantum numerics library, which provides implementations for a host of mathematical functions. It supports integer (signed & unsigned) and fixed-point representations.
+- [**Quantum machine learning library**](xref:microsoft.quantum.machine-learning.intro):
```

Comment:
```suggestion
- [**Quantum machine learning library**](xref:microsoft.quantum.machine-learning.concepts.intro):
```

Reply:
Thanks for the ping and for the suggestion, I think this got lost in the notifications pile. Happy to complete this now, as I think it's a significant improvement even as-is.